resources/views/service/vouchers/create.blade.php

```diff
@@ -1,48 +1,77 @@
 @extends('service.layouts.app')
 @section('content')
 
-<div id="container">
-    @include('service.includes.sidebar')
-    <div id="main-content">
-
-        <h1>Add a batch of voucher codes</h1>
-
-        <p>Use the form below to add a new batch of vouchers. Select an area, and then enter the starting and ending voucher code numbers.</p>
-
-        <form role="form" method="POST" action="{{ route('admin.vouchers.storebatch') }}" class="styled-form">
-            {!! csrf_field() !!}
-
-            <div class="horizontal-container">
-                <div class="select">
-                    <label for="sponsor_id">Area</label>
-                    <select name="sponsor_id" id="sponsor_id" class="{{ $errors->has('sponsor_id') ? 'has-error' : '' }}" required>
-                        <option value="">Please select an area</option>
-                        @foreach ($sponsors as $sponsor)
-                        <option value="{{ $sponsor->id }}" @if( old('sponsor_id') == $sponsor->id ) SELECTED @endif >{{ $sponsor->name }}</option>
-                        @endforeach
-                    </select>
-                    @if ($errors->has('sponsor_id')) <label for="sponsor_id" class="alert-danger">{{ $errors->first('sponsor_id') }}</label> @endif
-                </div>
+    <div id="container">
+        @include('service.includes.sidebar')
+        <div id="main-content">
 
-                <div>
-                    <label for="start" class="required">Starting voucher code</label>
-                    <input type="text" id="start" name="start" value="{{ old('start') }}" class="{{ $errors->has('start') ? 'error' : '' }}" required>
-                    @if ($errors->has('start')) <label for="start" class="alert-danger">{{ $errors->first('start') }}</label> @endif
-                    @if ($errors->has('start-serial')) <label for="start" class="alert-danger">{{ $errors->first('start-serial') }}</label> @endif
-                </div>
+            <h1>Add a batch of voucher codes</h1>
+
+            <p>Use the form below to add a new batch of vouchers. Select an area, and then enter the starting and ending
+                voucher code numbers.</p>
+
+            <form role="form" method="POST" action="{{ route('admin.vouchers.storebatch') }}" class="styled-form">
+                {!! csrf_field() !!}
```

Comment:
I don't think we need to escape it here or if we must, I'd use the @csrf directive instead 👀

Reply:
oh, that appears to have come in in 5.6... and we missed it. neat.
I'll make a card to modernise our blade